test/images/agnhost/guestbook/guestbook.go

```diff
@@ -77,19 +77,21 @@ func registerNode(registerTo, port string) {
 	}
 
 	hostPort := net.JoinHostPort(registerTo, backendPort)
-	_, err := net.ResolveTCPAddr("tcp", hostPort)
-	if err != nil {
-		log.Fatalf("--slaveof param and/or --backend-port param are invalid. %v", err)
-		return
-	}
 
 	request := fmt.Sprintf("register?host=%s", getIP(hostPort).String())
```

Comment:
It can still fail here, there's a fatal in ``getIP``, which causes it to fail instantly without any retries.

Reply:
Thanks for pointing that out! Will refactor